GUI/MainInstall.cs

```diff
@@ -99,8 +99,8 @@ public partial class Main
 
             GUI.user.RaiseMessage("About to install...\r\n");
 
-            // FIXME: here we should heat up the cache with any mods we're going to install, because 
-            // when this transaction types out it fucks up everything else.
+            // FIXME: here we should heat up the cache with any mods we're going to install, because
+            // when this transaction types out it messes up everything else.
```

Comment:
Thumbs down for the bowdlerisation

Reply:
I'll admit, I had to look up that reference; thanks for an interesting Wikipedia visit.

Responses:
1. Comments in the CKAN source code are not classic works of Shakespeare
2. The meaning of the comment is either the same (unless "to mess up" is an Americanism, unbeknownst to me), or clearer since it's not actually related to reproduction
3. The code of conduct expresses an intent to "‬help‭ ‬us‭ ‬create‭ ‬safe‭ ‬and‭ ‬positive‭ ‬experiences‭ ‬for‭ ‬everyone," and I would suggest that "everyone" includes people who find words I couldn't say on daytime TV off-putting

Are you completely opposed to any change here, or is a compromise possible?

Reply:
Nah, I'm cool with it. My Australian nature sees no problem with the language use, but I accept that some people may find it off-putting. This was more a comment than a request for change.

Reply:
Ah, in that case I'll count my blessings that you didn't call me a c*nt, and go about my business! :+1: